From 4ae8229bba4b8d2466113bec1c67db461924795a Mon Sep 17 00:00:00 2001
From: dru18 <dru41821@gmailcom>
Date: Mon, 20 May 2019 11:34:57 +0530
Subject: [PATCH 3/5] 	new file:   printemail.sh

---
 printemail.sh | 1 +
 1 file changed, 1 insertion(+)
 create mode 100755 printemail.sh

diff --git a/printemail.sh b/printemail.sh
new file mode 100755
index 0000000..d756bc0
--- /dev/null
+++ b/printemail.sh
@@ -0,0 +1 @@
+cd ~/git$USER/test2 && echo '==========' && git config -l | grep '.email' && echo '============' && cd ~/git$USER/test2 && echo '===============' && git config -l | grep '.email' && echo '============='
-- 
2.7.4

